File: Java/src/NapakalakiGame/Prize.java
/************************************
|                                   | 
|   Javier Izquierdo Vera           |
|   Juan Carlos Gallardo Morales    |
|   2ºBII                           |
|                                   |
|   NapakalakiGame - PDOO           |
|                                   |
************************************/

package NapakalakiGame;

public class Prize {
    private int treasures, levels;
    
    // Constructores
    public Prize(int level, int treasures){
        this.treasures = treasures;
        this.levels = level;
    }
    
    
    // Obtener atributos:
    public int getLevels(){
        return levels;
    }
    
    public int getTreasures(){
        return treasures;
    }
    
    public String toString(){
        return "Niveles: " + this.levels + " Tesoros: " + treasures;
    }

}

File: Ruby/lib/prize.rb
#encoding: utf-8
=begin
/***********************************/
|                                   | 
|   Javier Izquierdo Vera           |
|   Juan Carlos Gallardo Morales    |
|   2ºBII                           |
|                                   |
|   NapakalakiGame - PDOO           |
|                                   |
/***********************************/
=end


module NapakalakiGame
class Prize
  #Variables
  attr_reader:treasures, :levels
  
  #Constructor
  def initialize(treasures,level)
    @treasures = treasures # Tesoros que se ganan
    @levels = level # Nivel que se gana
  end
  
  def to_s
        return "\nPRIZE--> Treasures:  #{treasures} , Levels: #{levels}"
  end
  
end
end